draft-ietf-tls-dtls13.md

```diff
@@ -300,6 +307,44 @@ length:
 encrypted_record:
 : Identical to the encrypted_record field in a TLS 1.3 record.
 
+{:br}
+
```

Comment:
At this point I would point out that you can include multiple records in the same datagram.